package.json

```diff
@@ -13,8 +16,34 @@
   "dependencies": {
     "cheerio": "^0.22.0",
     "entities": "^1.1.1",
-    "micro": "^7.3.0",
+    "express": "^4.15.2",
     "request": "^2.81.0",
     "unquote": "^1.1.0"
+  },
+  "devDependencies": {
+    "axios": "^0.16.1",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "bootstrap-loader": "^2.1.0",
+    "bootstrap-sass": "^3.3.7",
+    "css-loader": "^0.28.1",
+    "date-fns": "^1.28.4",
+    "extract-text-webpack-plugin": "^2.1.0",
+    "file-loader": "^0.11.1",
+    "html-webpack-plugin": "^2.28.0",
+    "imports-loader": "^0.7.1",
+    "jquery": "^3.2.1",
+    "node-sass": "^4.5.2",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "resolve-url-loader": "^2.0.2",
+    "sass-loader": "^6.0.4",
+    "style-loader": "^0.17.0",
+    "url-loader": "^0.5.8",
+    "webpack": "^2.5.1",
+    "webpack-dev-server": "^2.4.5"
```

Comment:
Wasn't sure myself since I understood you that only dependencies which are needed to run the app in production need to be in dependencies and since those are bundled during build time they are not needed to run the app?